src/visual.ts

```diff
@@ -2016,44 +2026,48 @@ export class Visual implements IVisual {
         this.chart.selectAll(Visual.Dot.selectorName).remove();
     }
 
-    private static parseSettings(dataView: DataView, colorHelper: ColorHelper): PulseChartSettings {
-        const settings: PulseChartSettings = PulseChartSettings.parse<PulseChartSettings>(dataView);
+    private updateSettings(): void {
+        // if (typeof this.visualSettings.popup.fontSize.value === "string") {
```

Comment:
?

Reply:
Is it needed?

Reply:
You're right, we don't need it. In the code I wrongfully set IEnumMember[] options to use strings instead of numbers, which caused the value to become `null`. Fixed it.